core/trino-parser/src/main/antlr4/io/trino/sql/parser/SqlBase.g4

```diff
@@ -42,6 +42,11 @@ statement
     : query                                                            #statementDefault
     | USE schema=identifier                                            #use
     | USE catalog=identifier '.' schema=identifier                     #use
+    | CREATE CATALOG (IF NOT EXISTS)? qualifiedName
```

Comment:
I'm curious if SQL/MED syntax could be used instead. I'm not sure if it's part of the sql standard or some extension. The downside is it's using different terms, like servers instead of catalogs, which would be pretty confusing :
```
 CREATE SERVER catalog_name
   FOREIGN DATA WRAPPER connector_name
   OPTIONS ( not.sure 'if dots are OK in keys' );
```
Cc @martint